src/docdb/tree/DocDBDocumentsTreeItem.ts

```diff
@@ -85,7 +85,7 @@ export class DocDBDocumentsTreeItem extends DocDBTreeItemBase<ItemDefinition> {
         }
         const keyPath = partitionKey.split('/');
         let i: number;
-        for (i = 0; i < keyPath.length - 1; i++) {
+        for (i = 0; i <= keyPath.length - 1; i++) {
```

Comment:
This is why I always prefer `for...const...of` instead so you can't have off-by-one errors. Something like this:
```typescript
const properties = partitionKey.split('/');
for (const prop of properties) {
    if (interim.hasOwnProperty(prop)) {
        interim = interim[prop];
```